sync_issues_to_jira/requirements.txt

```diff
@@ -1,2 +1,2 @@
-jira>=3.0.1<3.1.0
+jira==3.6.0
 PyGithub>=1.55<1.56
```

Comment:
The next line `PyGithub` may have the same issue

Reply:
@ydc-0 Thanks. Got this after PR was already merged, I fixed [directly to the master](https://github.com/espressif/github-actions/blob/master/sync_issues_to_jira/requirements.txt#L2) without PR.